editoast/migrations/2024-01-17-140850_rollingstock-add-supported-signaling-systems/up.sql

```diff
@@ -0,0 +1,16 @@
+-- Your SQL goes here
+
+ALTER TABLE rolling_stock
+ADD supported_signaling_systems jsonb NOT NULL DEFAULT ('["BAPR", "BAL"]');
+
+UPDATE rolling_stock
+SET supported_signaling_systems='["BAPR", "BAL", "TVM300"]'
+WHERE name IN ('1TGVA','1TGVOSE','1TGVSE','2TGVA','2TGVOSE','2TGVSE');
+
+UPDATE rolling_stock
+SET supported_signaling_systems='["BAPR", "BAL", "TVM430"]'
+WHERE name IN ('1TGVDUPL');
+
+UPDATE rolling_stock
+SET supported_signaling_systems='["BAPR", "BAL", "TVM300", "TVM430"]'
+WHERE name IN ('1PSE300','1TGVPBA','1TGVDASY','2PSE300','2TGVPBA','2TGVDASY','1ICE3','1TGV2N2','1TGVPBKA','1TGVPOS','1TGVR','1TGVR350','1TGVR360','2TGV2N2','2TGVDUPL','2TGVPBKA','2TGVPOS','2TGVR','2TGVR360','BR407 US','BR407UM2','TGVAGT','TMSTEURO','VELARO E');
```

Comment:
I don't think this is appropriate: this data isn't public, and thus the migration shouldn't be either.

it's best to run this migration manually, on production systems where keeping the data is required, and just change the proprietary the import scripts. Keep this SQL somewhere, and run it manually.

Reply:
done